tools/x-repo-types/Makefile

```diff
@@ -1,38 +1,51 @@
-all: goal-v-sdk goal-v-spv
+all: clean goal-v-sdk goal-v-spv
+
+clean:
+	rm x-repo-types
+
+x-repo-types:
+	go build
 
 # go-algorand vs go-algorand-sdk:
 
 goal-v-sdk: goal-v-sdk-state-delta goal-v-sdk-genesis goal-v-sdk-block goal-v-sdk-blockheader goal-v-sdk-stateproof
 
-goal-v-sdk-state-delta:
+goal-v-sdk-state-delta: x-repo-types
 	x-repo-types --x-package "github.com/algorand/go-algorand/ledger/ledgercore" \
 		--x-type "StateDelta" \
 		--y-branch "develop" \
```

Comment:
@cce this is reasonable, but I'd also draw your attention to the `skip` and `skipReason` fields. The idea was that a long running project would decide to deal with it later, and leave a breadcrumb here with the rational:

https://github.com/algorand/go-algorand/blob/master/tools/x-repo-types/xrt_test.go#L55

Let's see how it plays out, we have lots of options in case this process becomes a problem.